pandas/tests/util/test_util.py

```diff
@@ -8,7 +8,9 @@
 
 import pytest
 from pandas.compat import intern
-from pandas.core.common import _all_none
+
```

Comment:
i am not going to comment on all the xtra lines, but pls clean up

Reply:
OK.  The habit (that I'll try to break) is to add lines between stdlib vs 3rd-party vs pandas-but-not-same-sub-package vs same-subpackage.